components/layout/table_row.rs

```diff
@@ -542,10 +538,6 @@ impl Encodable for CollapsedBorder {
 pub enum CollapsedBorderProvenance {
     FromPreviousTableCell = 6,
     FromNextTableCell = 5,
-    FromTableRow = 4,
-    FromTableRowGroup = 3,
-    FromTableColumn = 2,
-    FromTableColumnGroup = 1,
```

Comment:
If these are not used, there’s probably some parts of border collapsing that are not implemented yet. Please file an issue and add a FIXME comment that links to it here.

r=me with this